src/Symfony/Bridge/Doctrine/Security/RememberMe/DoctrineTokenProvider.php

```diff
@@ -89,12 +89,12 @@ public function updateToken(string $series, #[\SensitiveParameter] string $token
         $sql = 'UPDATE rememberme_token SET value=:value, lastUsed=:lastUsed WHERE series=:series';
         $paramValues = [
             'value' => $tokenValue,
-            'lastUsed' => $lastUsed,
+            'lastUsed' => \DateTimeImmutable::createFromInterface($lastUsed),
```

Comment:
this can even be skipped. When writing data to the database, both `Types::DATETIME_MUTABLE` and `Types::DATETIME_IMMUTABLE` accept the interface.

Reply:
right, updted